modules/mvt/src/lib/binary-vector-tile/features-to-binary.js

```diff
@@ -7,10 +7,12 @@
  * geoJSON. See `binary-vector-tile/VectorTileFeature` for
  * data format detais
  */
+import earcut from 'earcut';
 export function featuresToBinary(features, firstPassData, options = {}) {
   return fillArrays(features, firstPassData, {
     numericPropKeys: options.numericPropKeys || extractNumericPropKeys(features),
-    PositionDataType: options.PositionDataType || Float32Array
+    PositionDataType: options.PositionDataType || Float32Array,
+    triangulate: options.triangulate
```

Comment:
This needs to be added to the loader's documentation.

Reply:
As per discussion in deck.gl PR this option has now been removed and is always true (for the `binary` MVTLoader)